inc/saiport.h

```diff
@@ -3447,6 +3447,59 @@ typedef enum _sai_port_pool_attr_t
      */
     SAI_PORT_POOL_ATTR_QOS_WRED_PROFILE_ID,
 
+    /**
+     * @brief Enable DOT1P and CFI -> TC and COLOR MAP on port.
+     *
+     * Map id = #SAI_NULL_OBJECT_ID to disable map on port.
+     * Default no map.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_QOS_MAP
+     * @allownull true
+     * @validonly SAI_PORT_ATTR_QOS_DOT1P_TO_TC_MAP == SAI_NULL_OBJECT_ID and SAI_PORT_ATTR_QOS_DOT1P_TO_COLOR_MAP == SAI_NULL_OBJECT_ID
```

Comment:
those conditions are not valid, you cat use oid values, in validonly ccondition you can use attribute which is enum, bool or int/uint, but not oid